nucleus/dataset.py

```diff
@@ -857,8 +935,13 @@ def delete_annotations(
         Returns:
             Empty payload response.
```

Comment:
Silly quirk of Sphinx, we need to do the following to link to the imported member (by default, it goes to the module, which we don't want to expose)
```suggestion
            :class:`AsyncJob`: Empty payload response.
```